src/components/variant/ui/header.tsx

```diff
@@ -23,9 +28,25 @@ export const VariantHeader = observer(({ setLayout }: Props): ReactElement => {
   const history = useHistory()
   const location = useLocation()
 
-  const genInfo = get(toJS(variantStore.variant), '[0].rows[0].cells[0][0]', '')
+  const variant = toJS(variantStore.variant)
+  const rows: IAttributeDescriptors[] = get(variant, '[0].rows', [])
+
+  const genesRow = rows.find(element => element?.name === 'genes')
+  const variantWithoutGenesName = 'None'
+  const genes = genesRow?.cells[0][0] ?? variantWithoutGenesName
```

Comment:
`const findGenesInRow = () => {..}`
And move it outside the component function

Reply:
Moved